includes/Mapper/class-contactmapper.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Contact mapper
+ *
+ * @package OmnisendPaidMerbershipsProPlugin
+ */
+
+declare(strict_types=1);
+
+namespace Omnisend\PaidMembershipsProAddon\Mapper;
+
+use Omnisend\PaidMembershipsProAddon\Actions\OmnisendAddOnAction;
+use Omnisend\SDK\V1\Contact;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class ContactMapper
+ */
+class ContactMapper {
+	private const CUSTOM_PREFIX  = 'paid_memberships_pro';
+	private const CONSENT_PREFIX = 'paid_memberships_pro';
+
+	/**
+	 * Get Contact object
```

Comment:
functions do not fully reflect what they do

Reply:
Changed function name to reflect what they do